.github/workflows/ci.yml

```
@@ -306,7 +307,8 @@ jobs:
       
       # This will be a no-op under normal circumstances since the cluster would have been deployed
       # in deploy-cluster. It is needed in case we want to re-run the job after the cluster has been reaped.
-      - uses: realm/ci-actions/mdb-realm/deployApps@3f810b2d04e9dada2bde0b33ec90102e52a0b30a
+      - name: Try to create cluster
```

```suggestion
      - name: Create cluster
```

I feel using the `try to` wording is superficial - every step here is trying to do something and there's no reason to explicitly state that the attempt may fail.

---

Yes, it is with "try" because the cluster may have been already created.

---

Done.